common-lib/src/cmr/common/util.clj

```diff
@@ -276,6 +276,18 @@
           m
           m))
 
+(defn remove-non-empty-keys
```

Comment:
We should add unit tests for this function. 

The name is a little confusing to me. This is just like the `select-keys` core function except you just want to select keys with empty values. What about calling it `select-empty-keys`?

Reply:
Yeah, I was thinking about the same thing. Just waiting for a good name :)  Based on your suggestions about blank, I think I'm going to call it select-blank-keys. I think you will agree with that because I'm selecting out the nil or empty or white space valued keys.

Reply:
Done.